proto/p4/v1/p4runtime.proto

```diff
@@ -284,6 +284,24 @@ message Action {
 
 message ActionProfileActionSet {
   repeated ActionProfileAction action_profile_actions = 1;
+  // Added in v1.5.0.
+  enum ActionSelectionMode {
+    DEFAULT_MODE_DETERMINED_BY_ACTION_SELECTOR = 0;
+    // Uses the hash mode specified by the action selector.
```

Comment:
Would be good to clarify what happens if there is none (e.g. if it doesn't specify a hash)... Maybe "... or a target-defined hash mode if there is none."